.gitignore

```diff
@@ -97,7 +97,7 @@ local.properties
 ## Plugin-specific files:
 
 # IntelliJ
-/out/
+out/
```

Comment:
I had files showing up that should be ignored because of intellij. I can move this to a different PR if you really want but I figured it wasn't worth the overhead. (It's in its own commit).